.vscode/launch.json

```diff
@@ -66,7 +66,16 @@
       "mode": "debug",
       "program": "${workspaceRoot}/cli/cmd/turbo",
       "cwd": "${workspaceRoot}/examples/basic",
-      "args": ["run", "build"]
+      "args": ["run", "build", "--dry-run", "-vvv"]
```

Comment:
Should this be edited? Seems like the next task is doing the same thing

Reply:
Good catch! Will updated